doc/user_guide/signal1d.rst

```diff
@@ -121,8 +121,25 @@ Spike removal
 --------------
 
 :py:meth:`~._signals.signal1d.Signal1D.spikes_removal_tool` provides an user
-interface to remove spikes from spectra.
+interface to remove spikes from spectra. It is possible to use this tool
+on a specific interval of the data by :ref:`slicing the data
+<signal.indexing>`. For example, to keep the signal between indexes 8 and 17:
```

Comment:
I think "keep" is wrong, perhaps "to work only on the signal between indexes 8 and 17"